go.mod

```diff
@@ -12,13 +12,14 @@ require (
 	github.com/openshift/api v0.0.0-20220325173635-8107b7a38e53
 	github.com/openshift/client-go v0.0.0-20211209144617-7385dd6338e3
 	github.com/openshift/library-go v0.0.0-20220121154930-b7889002d63e
+	github.com/openshift/machine-api-provider-powervs v0.0.2-0.20220404072532-0bd184db2e40
```

Comment:
This dependency then becomes a dependency of all other providers, which makes bumping the deps a nightmare.

If you want defaulting for the powervs provider spec, it must first be moved into `openshift/api` with the other platforms

Reply:
Got it, I hope you mean adding the [`PowerVSMachineProviderConfig`](https://github.com/Karthik-K-N/machine-api-provider-powervs/blob/f8d06b4337415daa86e56295c297fef0e615575d/pkg/apis/powervsprovider/v1alpha1/powervsmachineproviderconfig_types.go#L31) to [machine/v1beta1](https://github.com/openshift/api/tree/master/machine/v1beta1)

Reply:
@Karthik-K-N lets submit a code for the review :)

Reply:
Please make sure to review the API conventions I linked in my previous comment and try to ensure your code meets the requirements before you submit, it will make the process easier